.github/workflows/docker-deploy.yml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2022, Proofcraft Pt Ltd
+#
+# SPDX-License-Identifier: BSD-2-Clause
+
+# Build and deploy standard set of docker containers
+
+name: Deploy
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+  schedule:
+    - cron: "3 17 * * 4"
```

Comment:
What is the natural language version of this cron expression?

Reply:
Every Thu (I think) on 17:03, i.e. once a week at a random non-full hour time.

Reply:
I should probably add that as a comment